src/main/java/org/terasology/metalrenegades/world/dynamic/MRBiome.java

```diff
@@ -23,29 +26,47 @@
     private final Name id;
     private final String displayName;
 
+    private SubSampledNoise strataNoise;
+
     private Block stone;
     private Block sand;
     private Block grass;
     private Block snow;
     private Block dirt;
+    /**
+     * All the rock types that can be used in strata.
+     * Strata always come in the same order, and wrap around at the end.
+     */
+    private Block[] rocks;
 
     MRBiome(String displayName) {
         this.id = new Name("MetalRenegades:" + name());
         this.displayName = displayName;
 
+        strataNoise = new SubSampledNoise(
+                new BrownianNoise(new SimplexNoise(2), 4),
```

Comment:
The `2` here is a hardcoded seed. That's obviously not a great idea, but the biome initializer doesn't currently have access to the seed - the biome initializer doesn't even run per-world. I'm not sure what the best solution is here, but it seems like a similar problem to one I just noticed where the game crashes if you open the world configuration dialog because the biome initializer is running before assets are available.